js/fc.js

```diff
@@ -39,6 +39,7 @@ var FAILSAFE_CONFIG;
 var RXFAIL_CONFIG;
 var PID_ADVANCED_CONFIG;
 var FILTER_CONFIG;
+var SPECIAL_PARAMETERS;
```

Comment:
The problem is that this comes at the cost of wasting one MSP command value, which sounds a bit silly if it's just for one version. Doing a 2.9.1 maintenance release that brings the MSP version in line with what 3.0.0 is sounds like a better plan.